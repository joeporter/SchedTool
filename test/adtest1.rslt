Hyperperiod 2000 ms

P1/C1_0 0
P1/C2_0 1000

