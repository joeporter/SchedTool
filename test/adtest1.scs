Resolution 1us

% Test a full schedule on a single processor

Proc P1 100MHz 0s 0s
Comp C1 =0.5Hz 1.0s
Comp C2 =0.5Hz 1.0s

