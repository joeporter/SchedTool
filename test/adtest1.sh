#!/bin/bash
echo ------------------------------------------------------
echo Running adtest1... full schedule on a single processor
SchedToolD -f adtest1.scs -o adtest1.tmprslt -b min
echo Comparing results
diff adtest1.rslt adtest1.tmprslt > adtest1.out
if [[ -s adtest1.out ]] ; then
echo ------------------------------------------------------
echo Differences:
cat adtest1.out
echo ------------------------------------------------------
echo Remember to delete adtest1.tmprslt and adtest1.out
echo  when you are finished debugging.
else
echo No differences found -- test successful.
rm -f adtest1.tmprslt adtest1.out
fi ;
echo ------------------------------------------------------

