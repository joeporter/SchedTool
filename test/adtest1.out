1,5c1,5
< Hyperperiod 2000 ms
< 
< P1/C1_0 0
< P1/C2_0 1000
< 
---
> Hyperperiod 2000 ms
> 
> P1/C1_0 0
> P1/C2_0 1000
> 
